### DISCUSSION TITLE
Adding webpage link into "alert_text"

### DISCUSSION THREAD

**Discussion by ngms17** [jertel/elastalert2#874] [Q&A] [answered]
Is it possible to add a link to a webpage inside the "alert_text" in order to use this link in the alert email?

Cheers

**Comment by jertel** [comment, marked as answer]
ElastAlert 2 supports automatically adding links to Kibana. For arbitrary links I expect it will work as well. Have you tried it?

**Reply by ngms17** [reply]
I want to add a link to an external tool (Evebox to be more precise). I have not tried but i am asking if it is possible

**Reply by ngms17** [reply]
Is there any guide i can follow? Any guide for arbitrary link as wel would be appreciated

**Reply by jertel** [reply]
https://elastalert2.readthedocs.io/en/latest/ruletypes.html#alert-content

```
alert_text: "My custom link: https://some.where.invalid"
alert_text_type: alert_text_only
```

**Reply by ngms17** [reply]
Thought that would be much more complex than this. Thank you!
